index.js

```diff
@@ -186,12 +191,17 @@ async function installAptPackages(packages) {
 
 async function installBinaryRelease({crystal, filePattern, path}) {
     if (crystal === Nightly) {
-        await IO.mv(await downloadCrystalNightly(filePattern), path);
+        await IO.mv(await downloadCrystalNightly(filePattern, "master"), path);
     } else {
-        if (crystal === Latest) {
-            crystal = null;
+        const version = BranchVersion.exec(crystal);
+        if (version) {
+            await IO.mv(await downloadCrystalNightly(filePattern, version[1]), path);
```

Comment:
Optional - seems like `else` could be eliminated?
With this edit and a bit more
```suggestion
            crystal = version[1]
```

Reply:
The `else` immediately following calls a different method (`downloadCrystalRelease`). Do you mean the `else` branch this is entirely sitting in?
I fear a bit more complexity for refactoring that to a branch with the combined condition `crystal === Nightly || BranchVersion.exec(crystal)`.

Reply:
Ah yes my bad, of course they're not the same function